database/migrations/2024_02_09_035707_create_students_table.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up()
+    {
+        Schema::create('students', function (Blueprint $table) {
+            $table->id('StudentID');
+            $table->string('Name', 100);
```

Comment:
Should we have a first name and last name column? easier to query if we ever need it?

Reply:
if we're only using one column for people's names, it's probably a good idea to call it "Full Name" so it includes both their first and last names. Having two columns should be a good idea to make it clearer and more organized.